src/Queue/Task/EmailTask.php

```diff
@@ -114,8 +114,14 @@ public function run(array $data, int $jobId): void {
 			$result = null;
 			try {
 				$mailer->setTransport($data['transport'] ?? 'default');
-				$content = isset($data['content']) ? $data['content'] : '';
-				$result = $mailer->deliver($content);
+
+				// Check if a reusable email should be sent
+				if (!empty($data['action']) && !empty($data['actionVars'])) {
+					$result = $mailer->send($data['action'], $data['actionVars']);
```

Comment:
Does it need actionVars? Or could it also be empty, only action required?
In that case I would only check for action key here.

Reply:
you are right, the actionVars are optional. I will remove the second check